cms/templates/admin/cms/page/tree/menu.html

```diff
@@ -76,7 +76,7 @@
         <div class="cms-tree-col">
             <div class="cms-tree-item cms-tree-item-preview
                 {# INFO: highlight active icon when in sidebar mode #}
-                {% if page.id|slugify == request.GET.page_id|slugify %} cms-tree-preview-active{% endif %}">
```

Comment:
One more thing: I guess the if statement does not need the `unlocalize`.

Reply:
Done